guides/common/modules/proc_updating-operating-system.adoc

```diff
@@ -0,0 +1,38 @@
+[id="updating-operating-system_{context}"]
+= Updating the operating system
+
+{Team} recommends performing operating system updates regularly, because the minor releases patch security vulnerabilities and minor issues discovered after code is released.
+
+If there are any released {ProjectServer} updates, those will be included when performing this update.
+We do not recommend updating the operating system independently of {ProjectServer}.
+Disconnected users should ensure that their installation is pointing at up to date {ProjectServer} repositories.
+
+.Prerequisites
+* Back up your {ProjectServer}.
+For more information, see {AdministeringDocURL}backing-up-{project-context}-server-and-{smart-proxy-context}_admin[Backing Up {ProjectServer} and {SmartProxyServer}] in _{AdministeringDocTitle}_.
+
+.Procedure
+. Use the health check option to determine if the system is ready for update.
+On first use of this command, `{foreman-maintain}` prompts you to enter the hammer admin user credentials and saves them in the `/etc/foreman-maintain/foreman-maintain-hammer.yml` file.
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-maintain} update check
+----
++
+Review the results and address any highlighted error conditions before performing the update.
+. Because of the lengthy update time, use a utility such as `tmux` to suspend and reattach a communication session.
+You can then check the update progress without staying connected to the command shell continuously.
++
+If you lose connection to the command shell where the update command is running, you can see the logged messages in the `{installer-log-file}` file to check if the process completed successfully.
```

Comment:
Technically people also should look at the maintain logs (and can re-attach to the tmux session).

But this is also a "bug" in the other chapters, why I was suggesting unification.